libut/uthash.pc.in

```diff
@@ -0,0 +1,9 @@
+prefix=/usr
+libdir=LIBDIR
+includedir=/usr/include
+
+Name: uthash
+Description: C macros for hash tables and more 
+Version: VERSION
+Libs: -L${libdir} -lut
+Cflags: -I${includedir}/uthash
```

Comment:
...no package manager requires it but autotools/cmakes...kind of these. Does debian has /usr/lib64/pkgconfig/glib-2.0.pc? this file is used by pkg-config to load a library's CFLAGS and LDFLAGS...so when you use such libraries, you don't have to find out them one by one on your own. reference: https://www.freedesktop.org/wiki/Software/pkg-config/